apps/prairielearn/src/ee/lib/aiQuestionGeneration.sql

```diff
@@ -28,3 +28,74 @@ FROM
 WHERE
   doc_path = $doc_path
   AND chunk_id = ANY ($chunk_ids);
+
+-- BLOCK update_draft_number
+UPDATE pl_courses
+SET
+  draft_number = draft_number + 1
+WHERE
+  id = $course_id
+RETURNING
+  draft_number;
```

Comment:
This would silently overwrite the first draft question- the fix that I have in mind is to update draft_number to max(draft_number, max of draft #s + 1) on sync, but IDK if there's a better option